app/src/main/java/org/fossasia/pslab/communication/peripherals/MCP4728.java

```diff
@@ -64,9 +69,8 @@ public double setVoltage(String name, int v) {
         return setRawVoltage("name", v);
     }
 
-    public int getVoltage(String name) {
-        // todo : add method when resolved in pslab-python
-        return -1;
+    public Double getVoltage(String name) {
+        return this.values.get(name);
```

Comment:
It should be double. During sciencelab porting this method wasn't resolved, so that created the problem of different types.
Would update in sciencelab in next PR.